Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.13.1
+
+#ENV GO111MODULE=on
+#ENV GOPROXY=https://proxy.golang.org
+#ENV GOPROXY=direct
+
+ADD go.mod /go/src/github.com/minio/m3/go.mod
+ADD go.sum /go/src/github.com/minio/m3/go.sum
+WORKDIR /go/src/github.com/minio/m3/
+# Get dependancies - will also be cached if we won't change mod/sum
+RUN go mod download
+
+ADD . /go/src/github.com/minio/m3/
+WORKDIR /go/src/github.com/minio/m3/
+WORKDIR /
+
+RUN CGO_ENABLED=0 GOOS=linux go build -ldflags "-s" -a -installsuffix cgo -o main ./go/src/github.com/minio/m3/
```

Comment:
```suggestion
RUN CGO_ENABLED=0 GOOS=linux go build -ldflags "-w -s" -a -installsuffix cgo -o main ./go/src/github.com/minio/m3/
```

Reply:
Why not add -o m3 and then use

Reply:
what does `-w` do?

Reply:
found out https://stackoverflow.com/questions/22267189/what-does-the-w-flag-mean-when-passed-in-via-the-ldflags-option-to-the-go-comman